src/components/Common/Navbar/Navbar.scss

```diff
@@ -0,0 +1,181 @@
+// Navbar.scss
+@import '../../../themes/mixins';
+@import '../../../themes/variables';
+
+.navbar-container {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 46px;
+
+  @include desktop {
+    height: 64px;
+    gap: 15.7px;
+    margin-bottom: 0px;
+    height: 46px;
+  }
+
+
+  .animated-search-bar {
+    display: flex;
+    align-items: center;
+    position: relative;
+    cursor: pointer;
+
+    .search-icon {
+      width: 24px;
+      height: 24px;
+
+    }
+
+    .search-input {
+      @include font-style;
+      width: 0;
+      opacity: 0;
+      border: none;
+      background-color: transparent;
+      transition: width 0.3s, opacity 0.3s, transform 0.3s;
+      font-size: 16px;
+      color: #333;
+      outline: none;
+      transform-origin: right;
+      transform: translateX(-150%);
+    }
+
+    &:hover .search-input {
+      width: 160px;
+      opacity: 1;
+      transform: translateX(9);
+    }
+  }
+}
+
+.navbar-left {
+  display: flex;
+  align-items: center;
+}
+
+.navbar-hamburger-menu {
+  @include desktop {
```

Comment:
The desktop-specific styles are applied directly within the nested selectors. While this is functional, it can make the code harder to read and maintain. Consider using media queries at the root level of the stylesheet to separate desktop-specific styles more cleanly.